### ISSUE TITLE
NTR: D-2-hydroxyglutaric acid high in CSF (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8344]
New term request D-2-hydroxyglutaric acid high in CSF (elthat_130528183532): Cerebrospinal fluid

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000246
- label: Elevated CSF D-2-hydroxyglutaric acid concentration
- parents: [Abnormal CSF carboxylic acid concentration (HP:0500183)](https://hpo.jax.org/app/browse/term/HP:0500183)
- pmid: PMID:20020533
- def: The concentration of D-2-hydroxyglutaric acid in the cerebrospinal fluid (CSF) is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8344
- synonyms:
